### PR TITLE
NAS-107709 / 20.10 / ES24f and ES102 support

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by darkfiberiru** [comment]
Looks good for es24f but doesn't seem to include es102 yet as indicated in PR

**Review by darkfiberiru — APPROVED** [review]
Looks good to me. All ses details for slots will be passed 1 to 1 via api?

**Review by william-gr — APPROVED** [review]

File: src/freenas/usr/local/lib/middlewared_truenas/alert/source/license_status.py [hunk 1]
```diff
@@ -189,15 +189,15 @@ def check_sync(self):
                             )
                         ))
 
-                # ES24F Expansion shelf
```
Comment (darkfiberiru) [inline src/freenas/usr/local/lib/middlewared_truenas/alert/source/license_status.py:192]: @themylogin This is correct as  ES24F sorry for typo in original ticket. This is via Andrew Nyguen Product PM who caught my mistake in ticket

**Review by darkfiberiru — CHANGES_REQUESTED** [review]
ES24F was correct typo in ticket. Shelves are always all caps.

**Review by darkfiberiru — APPROVED** [review]
Looks good now thank you

**Pull request merged**